modules/core/output_modules.php

```diff
@@ -1299,11 +1299,11 @@ class Hm_Output_main_menu_content extends Hm_Output_Module {
     protected function output() {
         $res = '';
         $email = false;
-        $single = $this->get('single_server_mode');
         if (array_key_exists('email_folders', merge_folder_list_details($this->get('folder_sources', array())))) {
             $email = true;
         }
-        if (!$single) {
+        $total_accounts = count($this->get('imap_servers', array())) + count($this->get('feeds', array()));
```

Comment:
This is for Everything Menu: check both(Email and Feeds)

Reply:
What about jmap_servers and why it is > 2 - if you have 2, everything will not be displayed which seems wrong.

Reply:
I checked jmap servers are not in: jmap_servers but imap_servers

Reply:
Ah, yes, forgot about that.